server/services/timelineService.ts

```diff
@@ -43,12 +43,15 @@ export default class TimelineService {
 
     // Lookup the prison for the first event by itself to prevent a race condition on the `PrisonService`
     // If the `PrisonStore` cache is empty/stale, attempting to lookup all the event's prisons in a `Promise.all`
-    // loop results in a race condition where all prison look ups run (essentially) at the same time, and they all try
+    // loop results in a race condition where all prison look-ups run (essentially) at the same time, and they all try
```

Comment:
IntelliJ was prompting me to fix this. I tried to ignore it, but failed! 😂

Reply:
🤣